rules/windows/builtin/win_mal_creddumper.yml

```diff
@@ -18,15 +18,17 @@ logsource:
 detection:
     selection:
         EventID: 7045
-    keywords:
-        Message:
-          - '*WCE SERVICE*'
-          - '*WCESERVICE*'
-          - '*DumpSvc*'
-    quarkspwdump:
-        EventID: 16
-        HiveName: '*\AppData\Local\Temp\SAM*.dmp'
-    condition: ( selection and keywords ) or quarkspwdump
+        ServiceName:
+            - '*WCE SERVICE*'
+            - '*WCESERVICE*'
+            - '*DumpSvc*'
+    malsvc_paexec:
+        ServiceFileName|contains: '\PAExec'
+    malsvc_wannacry:
+        ServiceName: 'mssecsvc2.0'
+    malsvc_persistence:
+        ServiceFileName|contains: 'net user'
+    condition: selection and 1 of malsvc_*
```

Comment:
I suggest to delete this lines and left them as part of `rules/windows/builtin/win_mal_service_installs.yml` (that was deleted) as they are not correlate with the name and description of this Sigma rule:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L1-L2

Reply:
Ok. my edit had the intent to consolidate similar stuff. But your suggestions provide a clearer separation of different topics.